Components/Molecules/Confetti.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ConfettiCannon from "react-native-confetti-cannon";
+import { Dimensions } from "react-native";
+
+const windowWidth = Dimensions.get("window").width;
+const Confetti = React.forwardRef(function passRefToChild(props, ref) {
+  return (
+    <ConfettiCannon
+      count={200}
+      origin={{ x: windowWidth / 2, y: -15 }}
+      autoStart={false}
+      ref={ref}
+      fadeOut={true}
+      fallSpeed={5000}
+    />
```

Comment:
Could we update the colors parameter to reflect our app's color scheme? Based on [this design](https://www.figma.com/file/Xp9oLxrMyh3TOz81ztHH5x/Push-Pirates?node-id=0%3A1999), palette is black, white grey, and blue. You can find the hex values in that figma too

Reply:
@ChrisCruze I added the colors from Figma. Check it out. Personally, I kinda like the original random colors better. But, then it may look better after we merge the new color updates to the buttons and the header. 

If that feature is ready to be merged, we can merge that real quick and I can import that change to my current branch to see it for real.